### PR TITLE
fix(TDI-43915): remove encryption lib from bundle

### PR DESCRIPTION
**Pull request by Voievidko**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-43915

**What is the new behavior?**
`com.couchbase.client.encryption` lib was removed from bundle.
After this fix Studio will trigger user to download it.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/325/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/325/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Arvoievidko%2FTDI-43915_Exclude_encryption_dep)

File: components/components-couchbase/pom.xml [hunk 1]
```diff
@@ -269,13 +269,13 @@
                                 <Require-Capability>osgi.ee;filter:="(&amp;(osgi.ee=JavaSE)(version=1.7))"</Require-Capability>
                                 <Bundle-SymbolicName>org.talend.components.couchbase</Bundle-SymbolicName>
                                 <Service-Component>*</Service-Component>
-                                <Import-Package>!org.apache.hadoop.*, !org.apache.http.*, !com.fasterxml.jackson.databind.*,
+                                <Import-Package>!com.couchbase.client.encryption.*, !org.apache.hadoop.*, !org.apache.http.*, !com.fasterxml.jackson.databind.*,
```
Comment (gonchar-ivan) [inline components/components-couchbase/pom.xml:272]: If we have some desing time action (like guess schema, get module list etc) for this component we will need to test it carefully
import package defines some kind of "classpath" for design time actions.

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/326/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/326/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/327/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/327/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/328/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/328/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Arvoievidko%2FTDI-43915_Exclude_encryption_dep)

**Pull request merged**